libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -139,9 +183,52 @@ void DocStringAnalyser::handleCallable(
 	StructurallyDocumentedAnnotation& _annotation
 )
 {
-	static set<string> const validTags = set<string>{"author", "dev", "notice", "return", "param"};
+	static set<string> const validTags = set<string>{"author", "dev", "notice", "return", "param", "inheritdoc"};
 	parseDocStrings(_node, _annotation, validTags, "functions");
 	checkParameters(_callable, _node, _annotation);
+
+	if (CallableDeclaration const* baseFunction = resolveInheritDoc(_callable.annotation().baseFunctions, _node, _annotation))
+		copyMissingTags(_annotation, baseFunction->annotation());
```

Comment:
Does `copyMissingTags` depend on the order in which the contracts are defined in the source file?
What about: A inherits from B inherits from C
```
contract A is B{}
contract B is C {}
contract C {}
```
And `B` explicitly inherits the docs from `C`, but `A` implicitly inherits the docs from `B`?

Reply:
Yes it depends on the order. But your example doesn't work:
```
Error: Definition of base has to precede definition of derived contract
 --> test.sol:1:15:
  |
1 | contract A is B{}
  |               ^


```

Reply:
Ah, so you are lucky! :)

Reply:
Can you still add a test where the middle contract inherits explicitly and the most derived inherits implicitly?